tests/test_unit/test_utils.py

```diff
@@ -0,0 +1,51 @@
+import glob
+import tempfile
+
+import pytest
+import yaml
+
+from wazp.utils import df_from_metadata_yaml_files
+
+
+def get_sample_project_metadata_fields() -> dict:
+    """Get the metadata dictionary from the sample project for testing."""
+    with open("sample_project/metadata_fields.yaml") as fi:
+        metadata_fields = yaml.safe_load(fi)
+    return metadata_fields
+
+
+def test_df_from_metadata_yaml_sample_project_metadata() -> None:
+    """Normal operation: test we can read the sample project metadata."""
+    fields = get_sample_project_metadata_fields()
+    testme = df_from_metadata_yaml_files("sample_project/videos", fields)
+
+    expected = set(fields)
+    actual = set(testme.columns)
+    diff = expected.symmetric_difference(actual)
+    assert expected == actual, f"Metadata fields -> df problem with: {diff}"
```

Comment:
I'd rephrase this a bit:
f"Metadata fields and df columns differ in the following fields: {diff}"

Or maybe more explicitly:
f"The following fields appear only in either the metadata fields or the df columns: {diff}"